crate_universe/Cargo.toml

```diff
@@ -6,7 +6,7 @@ members = [
 
 [package]
 name = "cargo-bazel"
-version = "0.6.0"
+version = "0.8.0"
```

Comment:
Odd, I thought there was a test for keeping these in sync. https://github.com/bazelbuild/rules_rust/issues/1883